packages/elements/src/photon-multirx-form/components/AddPrescriptionCard.tsx

```diff
@@ -221,7 +221,7 @@ export const AddPrescriptionCard = (props: {
         >
           <Show when={props.enableNewMedicationSearch}>
             <div class="mb-2">
-              <Banner status="info" withoutIcon closable name="new-medication-search-banner">
+              <Banner status="info" withoutIcon closable id="new-medication-search-banner">
```

Comment:
lol now that i think about it is `id` a reserved property? it hink it might be

Reply:
naw I think it's all good imo, I think it's still idiomatic